modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/DateProcessor.java

```diff
@@ -111,24 +111,17 @@ String getTargetField() {
 
     public static final class Factory implements Processor.Factory {
 
-        @SuppressWarnings("unchecked")
         public DateProcessor create(Map<String, Processor.Factory> registry, String processorTag,
                                     Map<String, Object> config) throws Exception {
             String field = ConfigurationUtils.readStringProperty(TYPE, processorTag, config, "field");
             String targetField = ConfigurationUtils.readStringProperty(TYPE, processorTag, config, "target_field", DEFAULT_TARGET_FIELD);
             String timezoneString = ConfigurationUtils.readOptionalStringProperty(TYPE, processorTag, config, "timezone");
             DateTimeZone timezone = timezoneString == null ? DateTimeZone.UTC : DateTimeZone.forID(timezoneString);
             String localeString = ConfigurationUtils.readOptionalStringProperty(TYPE, processorTag, config, "locale");
+            // TODO: default to Locale.ROOT?
```

Comment:
Is there a reason not to change it now? I think ROOT makes sense, but maybe I'm also missing sth. Who would know?

Reply:
we had a test that the default locale was english so I did not dare to change it. @talevy do you have information about this?

Reply:
I don't remember a specific rationale for this at the moment. I believe it is good to change to `ROOT` now